platform/app/src/components/ViewportGrid.tsx

```diff
@@ -173,6 +183,26 @@ function ViewerViewportGrid(props: withAppTypes) {
             'ViewportGrid::Unable to navigate to viewport containing',
             referencedDisplaySetInstanceUID
           );
+
+          try {
+            const currentViewport = cornerstoneViewportService.getCornerstoneViewport(viewportId);
+            const { viewPlaneNormal } = currentViewport.getCamera();
+            const referencedImageId = csToolsUtils.getClosestImageIdForStackViewport(
+              currentViewport as StackViewport,
+              measurement.points[0],
+              viewPlaneNormal
+            );
+            const imageIndex = (currentViewport as StackViewport)
+              .getImageIds()
+              .indexOf(referencedImageId);
+            csToolsUtils.jumpToSlice(currentViewport.element, { imageIndex });
+          } catch (error) {
+            console.warn(
+              'ViewportGrid::Unable to jump to image based on measurement coordinate',
+              error
+            );
+          }
+
```

Comment:
This shoouldn't be here, cornerstoneviewport service should not get called in the platform/**/*

Reply:
Updated